index.ts

```diff
@@ -273,7 +273,10 @@ const helmet: Helmet = Object.assign(
 
 export default helmet;
 
-// !helmet-end-of-commonjs
+// !helmet-start-of-commonjs-exports
+exports = helmet;
+module.exports = helmet;
+// !helmet-end-of-commonjs-exports
```

Comment:
This works, but I think it'd be slightly clearer with "code fences" around different parts. Would we be able to do something like this?

```ts
// !helmet-start-of-commonjs-exports
exports = helmet;
module.exports = helmet;
// !helmet-end-of-commonjs-exports

// !helmet-start-of-esm-exports
export {
  contentSecurityPolicy,
  // ...
};
// !helmet-end-of-esm-exports
```

Maybe this is silly to ask for.

Reply:
I can definitely do this, and have done so in https://github.com/helmetjs/helmet/pull/345/commits/c8e887c97ff4155dd41853066fea7e34cdd5804a. In the outputted ESM `index.js` it now includes these "code fence" comments which seems a bit unnecessary.

Would you prefer:

1. Keep the ESM "code fence" and keep the "code fence" in the outputted `index.js` for distribution
2. Keep the ESM "code fence" but add additional code to `withEsmFile()` to remove these comments before transpiling to JS
3. Remove the ESM "code fence"

Reply:
I think my preference would be to have no code fences anywhere in the export, because (1) it might confuse people looking at the exported files (2) it slightly increases the package size.

Reply:
@EvanHahn was thinking the same thing!

Sticking with option 2 above I've committed https://github.com/helmetjs/helmet/pull/345/commits/cbdce8410224dec6a9252edb5fd6614ce68acc0e that now removes those lines from the build.